src/Microsoft.Fx.Portability/BreakingChange.cs

```diff
@@ -87,7 +87,7 @@ public int CompareTo(BreakingChange other)
 
         public override int GetHashCode()
         {
-            return Id.GetHashCode();
+            return Id?.GetHashCode() ?? 0;
```

Comment:
If the Id is null, is it better to use `base.GetHashCode()` or some thing else other than 0? In the case that there are multiple BreakingChanges where the Id is not set.